### ISSUE TITLE
current hypertable version not supported

### ISSUE THREAD

**Issue by florakarniav** [brianfrankcooper/YCSB#760]
Hello, 

Since Hypertable current version majorly differs from the last supported version by ycsb (0.9.7.2 still works with capistrano which is now replaced by ht cluster tool), I think it would be great to support the new Hypertable stable version(0.9.8.11). I tried changing the version in pom.xml but there is no entry for the latest version in clojars repo, so the build fails. Can there be a workaround for this?

Thank you in advance

**Comment by kruthar** [comment]
I don't know much about hypertable, but if the dependencies are not hosted anywhere that we can easily access via Maven then there is not a whole lot that can be done I think. We faced similar issues in the past with things like Mapkeeper. I think we tried setting up a custom repo to hold it but it was not maintained and now Mapkeeper is not present in builds due to unavailability of depedenecies: #181.
